lms/views/admin/course.py

```diff
@@ -53,6 +56,40 @@ def show(self):
 
         return {"course": course}
 
+    @view_config(
+        route_name="admin.courses.dashboard",
+        request_method="GET",
+        permission=Permissions.STAFF,
+    )
+    def course_dashboard(self):
```

Comment:
I guess the reason to have this intermediary URL is just to track the audit event, right?